env/HERA.env

```diff
@@ -119,10 +119,12 @@ elif [[ "${step}" = "snowanl" ]]; then
 
```

Comment:
Change line above (118) to:
`export APRUN_SNOWANL="${APRUN_default} --mem=0 --cpus-per-task=${NTHREADS_SNOWANL}"`